tools/files.js

```diff
@@ -982,16 +983,199 @@ _.extend(files.KeyValueFile.prototype, {
       lines.push(k + "=" + v);
     }
     var newdata = lines.join('\n') + '\n';
-    fs.writeFileSync(self.path, newdata, 'utf8');
+    files.writeFile(self.path, newdata, 'utf8');
   },
 
   _readAll: function () {
     var self = this;
 
-    if (fs.existsSync(self.path)) {
-      return fs.readFileSync(self.path, 'utf8');
+    if (files.exists(self.path)) {
+      return files.readFile(self.path, 'utf8');
     } else {
       return null;
     }
   }
 });
+
+/////// Below here are wrappers of fs.* and path.* functions
```

Comment:
Good point, I'll fix up the style guide about file paths and update this comment this week.